src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+public class Lottery extends ColorSupplier {

```

Comment:
```suggestion
public class Lottery {
    private ColorSupplier colorSupplier = new ColorSupplier();
```
Наслідування доцільне, коли у нас `is a` відношення, а тут ми не можемо сказати, що Lottery is a ColorSupplier так як вона не кольори генереть а кульки. Тому краще використати композицію